app/code/core/Mage/Directory/Model/Currency.php

```diff
@@ -333,8 +333,8 @@ public function getConfigDefaultCurrencies()
      */
     public function getConfigBaseCurrencies()
     {
-        $defaultCurrencies = $this->_getResource()->getConfigCurrencies($this, self::XML_PATH_CURRENCY_BASE);
-        return $defaultCurrencies;
+        $defaultCurrencies = (string) Mage::app()->getConfig()->getNode(self::XML_PATH_CURRENCY_BASE, 'default');
+        return [$defaultCurrencies];
```

Comment:
why not change only the `Mage_Directory_Model_Resource_Currency::getConfigCurrencies` to use config (which is cached) ?

Reply:
Agreed, there doesn't seem to be any good reason to directly query the config table so `getConfigCurrencies` could simply be:

```php
    /**
     * @deprecated
     */
    public function getConfigCurrencies($model, $path)
    {
        return (string) Mage::app()->getConfig()->getNode($path, 'default');
    }
```

Reply:
you can't do this `return explode(',', (string) Mage::app()->getConfig()->getNode($path, 'default'));`, bcs. you can customize settings per website/store scope